build/packages-template/bbb-config/after-install.sh

```diff
@@ -125,6 +125,11 @@ else
   sed -i 's/events {/worker_rlimit_nofile 10000;\n\nevents {/g' /etc/nginx/nginx.conf
 fi
 
+# symlink default bbb nginx config from package if it does not exist
+if [ ! -e /etc/bigbluebutton/nginx/include_default.nginx ] ; then
+  ln -s /usr/share/bigbluebutton/include_default.nginx /etc/bigbluebutton/nginx/include_default.nginx
```

Comment:
When testing locally I had to first create the directory `mkdir -p /etc/bigbluebutton/nginx`. 

```suggestion
  mkdir -p /etc/bigbluebutton/nginx
  ln -s /usr/share/bigbluebutton/include_default.nginx /etc/bigbluebutton/nginx/include_default.nginx
```